django-s3-file-field-client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "django-s3-file-field-client",
+  "version": "0.0.1",
```

Comment:
The pip package uses Git tags for its version. I expect that setting up a similar system for npm publishes will be significantly more work. In the meantime, should we hardcode this version to match the pip package, then perhaps only bump it when the client is updated, which I expect will be much less frequently than the pip package.